BridgeSDK/BridgeAPI/SBBAuthManagerInternal.h

```diff
@@ -73,6 +73,15 @@ extern NSString * const kSBBAuthResetAPI;
 @property (nonatomic, strong) id<SBBObjectManagerProtocol> objectManager;
 @property (nonatomic, strong) SBBUserSessionInfo *placeholderSessionInfo;
 
+// Keep track of whether an auth/reauth call to Bridge is currently in progress.
+@property (nonatomic, assign) BOOL authCallInProgress;
```

Comment:
Should these be readonly?

Reply:
They're "private" (internal header file not exposed in the module) so I wasn't worried about that... should I be?

Reply:
went ahead with the suggestion anyway--doesn't hurt and might prevent a hypothetical future obj-c runtime twiddler from getting themselves in trouble if nothing else.

Reply:
Yeah, when someone else eventually inherits this code, readonly tells that dev that this *should* only be accessible by the auth manager. Or if it is exposed as internal so that tests can muck with it, might add a comment to that effect? Don't know, pretty edge case. ;)